build_helpers/startSiteDevServer.sh

```diff
@@ -5,6 +5,7 @@ PATH=$(npm bin):$PATH
 rm -rf ./__site__
 rm -rf ./__site_prerender__
 ./build_helpers/buildAPIDocs.sh
+webpack --config "$PWD/site/webpack-client.config.js"
```

Comment:
Slightly unrelated to this PR, but I found out that some of the output files necessary for the `buildSiteIndexPages.sh` script below wasn't being generated in production mode. So we run webpack before calling the script to generate it.